opendds_cmake_module/CMakeLists.txt

```
@@ -1,12 +1,11 @@
 cmake_minimum_required(VERSION 3.5)
 
-project(opendds_cmake_module NONE)
+project(opendds_cmake_module CXX)
```

```suggestion
project(opendds_cmake_module)
```
You shouldn't need the CXX since there is not C/C++ code in this project.  NONE is more appropriate but should be fine without it.

---

There was something that didn't work when the project was set to `NONE`, but I forgot what it was, Mike might remember.

---

https://cmake.org/cmake/help/v3.10/command/project.html#command:project That may be true but I'm pretty sure you can just leave it off too. It does the same thing and is standard style in ROS packages.

---

It works now... - removed